Makefile

```diff
@@ -133,6 +134,13 @@ artifacts-linux:
 	GOOS=linux GOARCH=arm64 CC=aarch64-linux-gnu-gcc make clean binaries
 	$(TAR) -C _output/ -czvf _artifacts/lima-$(VERSION_TRIMMED)-Linux-aarch64.tar.gz ./
 
+.PHONY: artifacts-windows
+artifacts-windows:
+	mkdir -p _artifacts
+	GOOS=windows GOARCH=amd64 CC=x86_64-w64-mingw32-gcc make clean binaries
+	$(TAR) -C _output/ -czvf _artifacts/lima-$(VERSION_TRIMMED)-Windows-x86_64.tar.gz ./
```

Comment:
Probably no need to create tar.gz for Windows

Reply:
The tarballs are the default, on all platforms (same everywhere). They work fine on Windows too, when having a development environment installed (e.g. Git for Windows)

The .zip was just a fallback, for a user setup with minimal tools